plugins/AITagger/config.py

```diff
@@ -0,0 +1,9 @@
+API_BASE_URL = 'http://localhost:8000'
```

Comment:
Consider putting this in a UI setting if it's going to be a commonly changed value - plugin updates are not merged on the client

Reply:
This will never get changed on my end and is only there for the user if they wanted to change it